cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/aggregation/AggregationDefinition.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
```

Comment:
2022

Reply:
I started writing that class in 2021... 😅 Fixed.